collection.php

```diff
@@ -0,0 +1,39 @@
+<?php require('database.php'); ?>
+<html>
+<head>
+    <title>Collection</title>
+    <link rel="stylesheet" type="text/css" href="collection.css">
+</head>
+<body>
+<header>
+    <nav>
+        <img class="guitar" src = "https://w0286994blog.files.wordpress.com/2016/04/scute.jpg?w=676" alt="Guitarlogo"></li>
+        <h1 class="title">THE GUITAR COLLECTION!!!!!</h1>
+    </nav>
+</header>
+<form action="http://localhost:1234/collection/guitarscollection/form.php">
+<button type="submit">the pig wants more rock<br>ADD GUITAR!</button>
+</form>
+<div class="collection">
+    <?php
+    if (!empty($result)) {
+        foreach ($result as $guitar) {
+            echo '<div><img class="images" src="' . $guitar['img'] . '" alt="' . $guitar['model'] . '"/>
+        Model: ' . $guitar['model'] . '<br>Year: ' . $guitar['year'] . '<br>Type: ' . $guitar['type'] . '<br>' .
```

Comment:
avoid using `<br>` where possible. This would be better if it used divs or p tags to create the new lines